src/Plugins/ImageProcessor.Web/ImageProcessor.Web.Plugins.AzureBlobCache/AzureBlobCache.cs

```diff
@@ -381,11 +381,12 @@ public override void RewritePath(HttpContext context)
                         {
                             HttpResponse contextResponse = context.Response;
                             cachedStream.CopyTo(contextResponse.OutputStream);
+                            string contentType = response.ContentType;
 
                             // Mimetype can be null when returning from the cache.
                             ImageProcessingModule.SetHeaders(
                                 context,
-                                string.IsNullOrWhiteSpace(this.mimeType) ? contextResponse.ContentType : this.mimeType,
```

Comment:
There must be a way we can guarantee the correct type? I go to great pains to ensure that the correct type is sent to Azure.